src/figuresClasses.ts

```diff
@@ -1,19 +1,86 @@
 export interface Figure {
+  shape: string,
+  color: string,
```

Comment:
It's better to create types to define what values can be stored in shape and color properties